src/app/hm-admin/hm-admin.component.ts

```diff
@@ -1,23 +1,46 @@
 import {Component, Input, OnInit} from '@angular/core';
 import {HTTP_PROVIDERS} from '@angular/http';
+import {Route} from '@angular/router';
+import {MdButton} from '@angular2-material/button';
+import {MdIcon, MdIconRegistry} from '@angular2-material/icon';
 import {ConfigService} from './services/config/Config.service';
 import {CoreDefinitionService} from './services/core/CoreDefinition.service';
 import {SchemaService} from './services/schema/schema.service';
+import {HeaderComponent} from './header/index'
+import {ContentComponent, SidenavService} from './content/index'
+import {HomeComponent} from './+home';
+import { ListComponent } from './+list';
+
+export const HmAdminRoutes: Route[] = [
+  // @todo remove after useAsDefault or redirectTo implementation
+  new Route({path: '/', component: HomeComponent}),
+  new Route({path: '/home', component: HomeComponent}),
```

Comment:
What ? the Home component ? Sure, it could be replace by a custom dashboard and by a login page when we'll have one.